home/tests/test_insert_question_feature.py

```diff
@@ -154,3 +152,21 @@ def test_redirection_user_logged_out(self, client):
 
             assert response.status_code == 302
             assert response.url == '/login/?next=/explore/new_question'
+
+        @pytest.mark.django_db
+        def test_valid_tags_not_removed_from_question(self, client, authenticated_user):
+            data = {'title': "Question in Math",
+                    'subject': 1, 'content': '<p>a</p>',
+                    'grade': Grade.GRADE7}
+            client.post('/explore/new_question', data=data)
+            question = Question.objects.filter(title=data["title"]).first()
+            assert "<p>" in question.content
```

Comment:
@idokk1 
I think we want to remove also the valid tags so the content will look clean to the user (who probably does not know how to program and don't know what those tags mean )

Reply:
the valid tags are remove in the HTML page (thats the |safe purpose) ,
what i prevent here is that unwanted tags will execute on |safe , like @ifireball example